pom.xml

```diff
@@ -103,16 +98,15 @@
         <killbill-client.version>1.0.6</killbill-client.version>
         <!-- This should point to the latest non-SNAPSHOT version of commons -->
         <killbill-commons-for-tests.version>0.22.1</killbill-commons-for-tests.version>
-        <killbill-commons.version>0.22.1</killbill-commons.version>
-        <killbill-platform.version>0.38.2</killbill-platform.version>
+        <killbill-commons.version>0.22.2-SNAPSHOT</killbill-commons.version>
+        <killbill-platform.version>0.38.3-SNAPSHOT</killbill-platform.version>
```

Comment:
Do we need a snapshot ?

Reply:
This is only to make CircleCI happy. If a project (e.g. `killbill`) has a snapshot dependency on branch `feature-X`, we will pull and build locally dependencies (e.g. `killbill-platform` and `killbill-commons`) on branch `feature-X` (if it exists). I have a few branches `jdk8-upgrade` in other repos coming up 😄 

This is why https://github.com/killbill/killbill/pull/1052 is green for instance (there is a `killbill-oss-parent` `caching-improvements` branch).

Will cleanup before merging and releasing.